transform/snowflake-dbt/models/events/user_events_telemetry_precalc.sql

```diff
@@ -0,0 +1,241 @@
+        , TRIM(SPLIT_PART(SPLIT_PART(uet.feature_flags,',',2),':',2)) AS preparing_workspace_new_wording
+        , TRIM(SPLIT_PART(SPLIT_PART(uet.feature_flags,',',2),':',2)) AS sso
+        , uet.context_timezone AS context_timezone
+        , uet.context_device_id AS context_device_id
+        , uet.context_library_name AS context_library_name
+        , uet.context_screen_width AS context_screen_width
+        , uet.context_network_wifi AS context_network_wifi
+        , COALESCE(uet.user_actual_id, anonymous_id) AS user_actual_id
+        , uet.context_device_name AS context_device_name
+        , uet.user_id AS user_id
+        -- , COALESCE(NULLIF(${user_agent_registry.os_version}, 'Other'), uet.context_os_version) AS context_os_version
+        -- , CASE WHEN nullif(SPLIT_PART(${context_os_version}, '.', 2), '') IS NULL THEN ${context_os_version} ELSE SPLIT_PART(${context_os_version}, '.',1) || '.' || SPLIT_PART(${context_os_version}, '.',2) END AS os_version_major
+        , coalesce(uet.context_app_build, uet.context_traits_app_build) AS context_app_build
+        , uet.name AS name
+        -- Fixed typo - renamed to sign_up_sequence from sign_up_seqence
+        , CASE
+            WHEN COALESCE(uet.type, uet.event) IN ('cloud_signup_b_page_visit','cloud_signup_page_visit') THEN 1
+            WHEN COALESCE(uet.type, uet.event) = 'click_start_trial' THEN 2
+            WHEN COALESCE(uet.type, uet.event) = 'pageview_verify_email' THEN 3
+            WHEN COALESCE(uet.type, uet.event) = 'enter_valid_code' THEN 4
+            WHEN COALESCE(uet.type, uet.event) = 'pageview_create_workspace' AND uet.name = 'pageview_company_name' THEN 5
+            WHEN COALESCE(uet.type, uet.event) = 'workspace_name_valid' THEN 6
+            WHEN COALESCE(uet.type, uet.event) = 'workspace_provisioning_started' THEN 7
+            WHEN COALESCE(uet.type, uet.event) = 'workspace_provisioning_ended' THEN 8
+            ELSE NULL
+          END AS sign_up_sequence
+        , CASE
+            WHEN COALESCE(uet.type, uet.event) IN ('cloud_signup_b_page_visit','cloud_signup_page_visit') THEN 'Sign-Up Page'
+            WHEN COALESCE(uet.type, uet.event) = 'click_start_trial' THEN 'Click Start Trial'
+            WHEN COALESCE(uet.type, uet.event) = 'pageview_verify_email' THEN 'Verify Email'
+            WHEN COALESCE(uet.type, uet.event) = 'enter_valid_code' THEN 'Enter Valid Code'
+            WHEN COALESCE(uet.type, uet.event) = 'pageview_create_workspace' AND uet.name = 'pageview_company_name' THEN 'Enter Company Name'
+            WHEN COALESCE(uet.type, uet.event) = 'workspace_name_valid' THEN 'Workspace Name Valid'
+            WHEN COALESCE(uet.type, uet.event) = 'workspace_provisioning_started' THEN 'Workspace Creation Started'
+            WHEN COALESCE(uet.type, uet.event) = 'workspace_provisioning_ended' THEN 'Workspace Creation Ended'
+            ELSE NULL
+          END AS sign_up_events_sequence
```

Comment:
We can remove this field, as I am doing this calculation in Looker.

Reply:
That's the idea, not to have to do the calculation in Looker.

Reply:
No, this calculation is done in the visualization, after analysis. There is an option to add calculated field based on the output of the query, which is not part of the select statement.

Reply:
We can drop the field in Looker, to avoid confusion.

Reply:
the type and event we need to keep, the sign_up_events_sequence we can remove